hs_core/views/__init__.py

```diff
@@ -1024,30 +1024,26 @@ def create_resource(request, *args, **kwargs):
     if url_key in page_url_dict:
         return render(request, page_url_dict[url_key], {'title': res_title, 'metadata': metadata})
 
-    # try:
     resource = hydroshare.create_resource(
             resource_type=request.POST['resource-type'],
             owner=request.user,
             title=res_title,
             metadata=metadata,
             files=resource_files,
             fed_res_file_names=fed_res_file_names,
-            fed_res_path = fed_res_path[0] if len(fed_res_path)==1 else '',
+            fed_res_path = fed_res_path[0] if len(fed_res_path) == 1 else '',
             fed_copy_or_move=fed_copy_or_move,
             content=res_title
     )
-    # except Exception as ex:
-    #     context = {'resource_creation_error': ex.message }
-    #     return render_to_response('pages/create-resource.html', context, context_instance=RequestContext(request))
```

Comment:
@pkdash Why do you comment out try...except block here? I think try...except block is needed here.